chapters/vectors-matrix-ops/independence/reading/README.md

```diff
@@ -1 +1,162 @@
+
+Since the conditional probability of $A$ given $B$ is defined by:
+
+$$P [A|B] = \displaystyle\frac {P [A \cap B]} {P[B]}$$
+
+We see that $A$ and $B$ are independent if and only if
+
+$$P[A|B] = P[A] \quad (\text{when } P [B] > 0 )$$
+
+Two continuous random variables, $X$ and $Y$, are similarly independent if,
+
+$$P [X \in A, Y \in B] = P [X \in A] P[Y \in B]$$
+
+### Details
+
+Two continuous random variables, $X$ and $Y$, are similarly independent if,
+
+$$P [X \in A, Y \in B] = P [X \in A] P[Y \in B]$$
+
+Now suppose $X$ has `p.d.f.` $f_X$, and $Y$ has `p.d.f.` $f_Y$.
+Then,
+
+$$P [X \in A] = \int_{A} f_X (x) dx,$$
+
+$$P [Y \in B] = \int_{B} f_Y (y) dy$$
+
+So $X$ and $Y$ are independent if:
+
+$$P [X \in, Y \in B] = \int_{A} f_X (x) dx \int_{B} f_Y (y) dy$$
+
+$$= \int_{A}f_X (x) (\int_{B} f_Y (y) dy) dx$$
+
+$$= \int_{A}\int_{B} f_X (x)f_Y (y) dydx$$
+
+But, if $f$ is the joint density of $X$ and $Y$ then we know that
+
+$$P [X \in A, Y \in B]$$
+
+$$\int_{A}\int_{B} f (x,y) dydx$$
+
+Hence $X$ and $Y$ are independent if and only if we can write the joint density in the form of,
+
+$$f(x,y) = f_X (x)f_Y (y)$$
+
+## Independence and Expected Values
+
+If $X$ and $Y$ are independent random variables then $E[XY]=E[X]E[Y]$.
+
+Further, if $X$ and $Y$ are independent random variables then $E[g(X)h(Y)]=E[g(X)]E[h(Y)]$ is true if $g$ and $h$ are functions in which expectations exist.
+
+### Details
+
+If $X$ and $Y$ are random variables with a joint distribution function $f(x,y)$, then it is true that for $h:\mathbb{R}^2\to\mathbb{R}$ we have
+
+$$E[h(X,Y)]=\int\int h(x,y)f(x,y)dxdy$$
+
+for those $h$ such that the integral on the right exists
+
+Suppose $X$ and $Y$ are independent continuous random variables, then
+
+$$f(x,y) = f_X (x) f_Y (y)$$
+
+Thus,
+
+$$E[XY] = \int\int xy f (x,y) dxdy$$
+
+$$= \int\int xy f_X (x) f_Y (y) dxdy$$
+
+$$= \int xf_X (x) dx \int yf_Y (y) dy$$
+
+$$= E [X] E [Y]$$
+
+:::note Note
+
+Note that if $X$ and $Y$ are independent then $E[h(X) g(Y)] = E [h(X)] E[g(Y)]$ is true whenever the functions $h$ and $g$ have expected values.
+
+:::
+
+### Examples
+
+:::info Example
+
+Suppose $X,Y \in U (0,2)$ are independent identically distributed then,
+
+$$f_X (x) = \begin{cases} \displaystyle\frac{1}{2} & \text{if } 0 \leq x \leq 2 \\ 0 & \text{otherwise} \end{cases}$$
+
+and similarly for $f_Y$.
+
+Next, note that,
+
+$$f(x,y) = f_X (x) f_Y (y) = \begin{cases} \displaystyle\frac{1}{4} &\text{if } 0 \leq x,y \leq 2\\ 0 & \text{otherwise} \end{cases}$$
+
+Also note that $f(x,y) \geq 0$ for all $(x,y) \in \mathbb{R}^2$ and
+
+$$\int\int f(x,y)dxdy = \int_{0}^{2}\int_{0}^{2} \frac {1}{4} dxdy = \displaystyle\frac {1}{4}.4 = 1$$
+
+It follows that
+
+$$E \[X Y\] &= \_-\^\_-\^ f(x,y) xy dxdy\ > &= \_y=0\^2\_x=0\^2 xy dxdy\ > &= \_y=0\^2 y (\_x=0\^2 x dx) dy\ > &= \_y=0\^2 y \_x=0\^2 dy\ > &= \_y=0\^2 y ( \^2 - \^2 ) dy\ > &= \_0\^2 y dy\ > &= \_0\^2 y dy\ > &= y\^2 \| \_0\^2\ > &= \^2\ > &= 1$$
```

Comment:
```suggestion
$$\begin{aligned} E [X Y] &= \_-\^\_-\^ f(x,y) xy dxdy\ > &= \_y=0\^2\_x=0\^2 xy dxdy\ > &= \_y=0\^2 y (\_x=0\^2 x dx) dy\ > &= \_y=0\^2 y \_x=0\^2 dy\ > &= \_y=0\^2 y ( \^2 - \^2 ) dy\ > &= \_0\^2 y dy\ > &= \_0\^2 y dy\ > &= y\^2 \| \_0\^2\ > &= \^2\ > &= 1 \end{aligned}$$
```
This doesn't render properly and I don't understand what it's trying to say so I can fix it. This is the only place in the repo where `\^` and `\_` are present.

Reply:
I fixed it.